src/evict/evict_lru.c

```diff
@@ -448,8 +454,22 @@ __evict_server(WT_SESSION_IMPL *session, bool *did_work)
         return (0);
 
     __wt_epoch(session, &now);
-    if (WT_TIMEDIFF_SEC(now, cache->stuck_time) > WT_MINUTE * 5) {
-#if defined(HAVE_DIAGNOSTIC)
+
+#define WT_CACHE_STUCK_TIMEOUT_S 300
```

Comment:
I would change "_S" to "_SEC" so that it is obvious what the unit is.

Reply:
Replaced in 550153e69b95cdaf79a07ba8addc04325e23fe81